test/show.jl

```diff
@@ -2673,3 +2673,6 @@ end
 using .Issue49382
 (::Type{Issue49382.Type49382})() = 1
 @test sprint(show, methods(Issue49382.Type49382)) isa String
+
+#issue #52641
+@test Base.isidentifier(Symbol("e\u0301")) == false
```

Comment:
Add some more tests and comments:
```suggestion
@testset "isidentifier normalization" begin
    @test !Base.isidentifier(Symbol("e\u0301")) # not NFC-normalized
    @test Base.isidentifier(Symbol("\u00E9"))   # NFC-normalized
    @test !Base.isidentifier(Symbol("\u00B5))   # U+00B5 = µ normalized to U+03BC = μ by parser
    @test Base.isidentifier(Symbol("\u03BC"))   # U+03BC = μ is the normalized form
end
```